<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密码管理器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src/icons.qrc">
    <normaloff>:/mainui/icon/MainIcon.ico</normaloff>:/mainui/icon/MainIcon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {background:rgb(248, 248, 248);}

QToolTip{
	border:0px solid rgb(118, 118, 118); 
	background-color: #ffffff;
	color:#484848; 
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane {
    border-color: #cccccc;
}
QTabBar::tab {
    min-width:90px;
    color: #333333;
    border: 2px solid;
	padding:2px;
	border-radius:5px;
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.494737 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
	border-color:rgb(220, 220, 220);
	margin-right:2px;
}

QTabBar::tab:selected{
    /*min-width:120px;
    margin-left:50px;
    margin-right:50px;
    min-height:40px;*/
    color: #1296DB;
    border-top: 2px solid;
    border-color: white;
	border-top-color:#1296DB;
	background-color:white;
}
QTabBar::tab:hover {
	color:#1296DB;
}

QCheckBox {
    padding: 2px;margin: 2px;
	background:Transparent;
}
QCheckBox::indicator {
    width: 15px;
    height: 15px;
}
QCheckBox::indicator:unchecked {
    border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#34495E;
  	background-color:rgb(255,255,255);
}
QCheckBox::indicator:checked {
	border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#0078d7; 
	image: url(:/mainui/icon/selected.png);
}
QCheckBox::indicator:hover{
	border-color:#0078d7; 
}
QCheckBox:hover {
	color:rgb(0, 0, 216);
}

/*
 * SpinBox 
 */
QSpinBox {
	border-style:solid;
	border-width:2px;
	border-radius:6px;
	border-color:#34495E;
	padding-left:3px;
	padding-right:3px;
	margin: 1px;
}
QSpinBox:hover{
	color:rgb(0, 0, 216);
	border-color:#0078d7;
}
QSpinBox::up-button {
    padding:1px;
    background-color:transparent;
}
QSpinBox::down-button {
    padding:1px;
    background-color:transparent;
}
QSpinBox::down-button {
    padding:1px;
    background-color:transparent;
}
/* 向上箭头*/
QSpinBox::up-arrow {
	image:url(:/mainui/icon/up1.png);
	height: 15px;
}
QSpinBox::up-arrow:hover {
	
	image: url(:/mainui/icon/up0.png);
	height: 15px;
}
QSpinBox::up-arrow:pressed {
	
	image: url(:/mainui/icon/up0.png);
	height: 13px;
}
/*向下箭头*/
QSpinBox::down-arrow {
	image: url(:/mainui/icon/down1.png);
	height: 15px;
}
QSpinBox::down-arrow:hover {
	image: url(:/mainui/icon/down0.png);
	height: 15px;
}
QSpinBox::down-arrow:pressed {
	image: url(:/mainui/icon/down0.png);
	height: 13px;
}
/*
 * groupbox
 */
QGroupBox {
	background-color:#E5E5E5;
	border-color:#E5E5E5;
	border-radius:20px;
	padding:3px;
	margin:5px;
}
QGroupBox:hover {
	background-color:#e6e6e6;
	padding:5px;
}
QGroupBox::title {
	color: rgb(0, 0, 0);
	subcontrol-origin: margin;
	subcontrol-position: top left;
	left:10px;
	top:10px;
	padding: 0px;
}

QLabel {
	background:Transparent;
}
</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab0">
      <attribute name="icon">
       <iconset resource="src/icons.qrc">
        <normaloff>:/mainui/icon/mgr1.png</normaloff>
        <normalon>:/mainui/icon/mgr0.png</normalon>
        <selectedoff>:/mainui/icon/mgr0.png</selectedoff>:/mainui/icon/mgr1.png</iconset>
      </attribute>
      <attribute name="title">
       <string>密码管理</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLineEdit" name="le_filiter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	border-style:none;
	padding:3px;
	border-radius:11px;
	border:2px solid #DCE4EC;
	background:rgb(255, 255, 255);
}
QLineEdit:focus {
	border:2px solid #34495E;
	background:rgb(255, 255, 255);
}
</string>
           </property>
           <property name="placeholderText">
            <string>筛选</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Ignored</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">
QComboBox {
    border: 1px solid gray;   /* 边框 */
    border-radius: 3px;   /* 圆角 */
    padding: 1px 18px 1px 3px;   /* 字体填衬 */
    color:#FFFFFF;
    background:#34495E;
}
/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
	/* 选定项的虚框 */
    outline: 0px solid gray;
	/* 整个下拉窗体的边框 */   
    border: 1px solid white;   
    color:#FFFFFF;
    background:rgb(68, 96, 124);
}

</string>
           </property>
           <item>
            <property name="text">
             <string>全部</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>名称</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>账号</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>邮箱/电话</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>380</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbt_about">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>关于</string>
           </property>
           <property name="toolTipDuration">
            <number>0</number>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	padding:6px;
	border: 0px;
	background-color:transparent;
	image: url(:/mainui/icon/ab1.png);
}
QPushButton:hover
{
	border: 0px;
	background-color:transparent;
	image: url(:/mainui/icon/ab0.png);
}
QPushButton:pressed
{
	padding:6px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbt_setting">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>设置</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	padding:6px;
	border: 0px;
	background-color:transparent;
	image: url(:/mainui/icon/set1.png);
}
QPushButton:hover
{
	border: 0px;
	background-color:transparent;
	image: url(:/mainui/icon/set0.png);
}
QPushButton:pressed
{
	padding:6px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>删除项</string>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	padding:6px;
	border: 0px;
	background-color:transparent;
	image: url(:/mainui/icon/delete1.png);
}
QPushButton:hover
{
	border: 0px;
	background-color:transparent;
	image: url(:/mainui/icon/delete.png);
}
QPushButton:pressed
{
	padding:6px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbt_add">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>添加项</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	padding:6px;
	border: 0px;
	background-color:transparent;
	image: url(:/mainui/icon/add.png);
}
QPushButton:hover
{
	border: 0px;
	background-color:transparent;
	image: url(:/mainui/icon/add1.png);
}
QPushButton:pressed
{
	padding:6px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="shortcut">
            <string>Ctrl+N</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget {
    color:#34495E;
    background:#DCDCDC;
    border:1px solid #b6b6b6;
    alternate-background-color:#d2d2d2;
    gridline-color:#ffffff;
	font-family:'consolas,Microsoft YaHei';
	font-size:10pt;
}
QTableWidget::item{
    border-radius:6px;
}
QTableWidget::item:selected{
color:#ffffff;
background:#73A1D0;
}

QTableWidget::item:hover{
background:rgb(104, 200, 252);
color:#000000;
    font-size:10;
}
QHeaderView::section{
text-align:center;
background:#496684;
padding:2px;
margin:1px;
color:#ffffff;
border:1px solid #242424;
border-left-width:0;
border-radius:4px;
}
QScrollBar:vertical{
background:#484848;
padding:0px;
border-radius:6px;
max-width:12px;
}
QScrollBar::handle:vertical{
background:#CCCCCC;
}
QScrollBar::handle:hover:vertical,QScrollBar::handle:pressed:vertical{
background:#A7A7A7;
}
QScrollBar::sub-page:vertical{
background:444444;
}
QScrollBar::add-page:vertical{
background:5B5B5B;
}
QScrollBar::add-line:vertical{
background:none;
}
QScrollBar::sub-line:vertical{
background:none;
}
</string>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::CopyAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>140</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>28</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>40</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>id</string>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>名称</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>账号</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>密码</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>邮箱/电话</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>备注</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src/icons.qrc"/>
 </resources>
 <connections/>
</ui>
